service/k8sclient/common/core.go

```diff
@@ -58,6 +64,15 @@ type PodSpec struct {
 	InitContainers []ContainerSpec `json:"initContainers,omitempty"`
 }
 
+// PodStatus holds pod status.
+type PodStatus struct {
+	// ContainerStatuses holds container statuses.
+	ContainerStatuses []ContainerStatus `json:"containerStatuses,omitempty"`
+
+	// ContainerStatuses holds container statuses.
```

Comment:
Same comment? What InitContainers will hold?

Reply:
Good catch, I updated the comment.